config/cesm/machines/config_machines.xml

```diff
@@ -1597,6 +1597,12 @@ This allows using a different mpirun command to launch unit tests
       <!-- The following is needed in order to run qsub from the compute nodes -->
       <env name="PATH">$ENV{PATH}:/cluster/torque/bin</env>
     </environment_variables>
+    <environment_variables comp_interface="nuopc" mpilib="mvapich2" DEBUG="FALSE">
+      <env name="ESMFMKFILE">/scratch/cluster/himanshu/PROGS/esmf/master/mvapich2/2.3.3/intel/20.0.1/lib/libO/Linux.intel.64.mvapich2.default/esmf.mk</env>
```

Comment:
This uses a user path in a scratch file system.

Reply:
I'm not sure what to do about this - since I do not believe that Himanshu has moved these to another location on izumi.

Reply:
Where do you suggest that he create these libraries?